nebula-components/components/_btn.scss

```diff
@@ -47,4 +47,68 @@
   .#{$nb-namespace}c-btn[disabled] {
     opacity: 0.6;
   }
+
+  .#{$nb-namespace}c-btn-group {
+    position: relative;
+    //overflow: hidden;
+  }
+
+  .#{$nb-namespace}c-btn-group__dropdown-toggle {
+    width: 100%;
+
+    &:hover,
+    &:focus {
+      outline: none;
+    }
+
+    &::before {
+      $size: 4rem;
+      content: '';
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      width: $size;
+      @include btn-color(darken($nb-button-alpha-color, 10%));
+      background-image: url('/src/img/chevron-down.svg');
```

Comment:
Like described above, use the base64 image used on the select box.  Because it's repeated across several components gzip will optimise them nicely - it loves repetition.